docs/SDK/Explanation/WalletSigs/authSig.md

```diff
@@ -97,3 +97,62 @@ Below is the complete [**React** project](https://replit.com/@lit/Smart-Contract
 ## Clearing the stored AuthSig
 
 If you want to clear the authSig stored in the browser local storage, you can call the [`disconnectWeb3` method](https://js-sdk.litprotocol.com/functions/auth_browser_src.ethConnect.disconnectWeb3.html).
+
+## What happens to the Authsig submitted to the Lit nodes?
+
+When you submit the `authSig` for retrieving the `symmetricKey` using the function `getEncryptionKey` the nodes verify the `authSig` based on the `accessControlConditions` passed along with it:
```

Comment:
can we generalize this statement to cover more SDK methods? Do we need to be so specific here?

Reply:
I agree with Howard and I think this section goes too deep... I think a post on spark might be a better place to do a deeper dive on authSig and how it works with Lit than the developer docs.

Reply:
can we put it in like an "advanced" section or something?  or move to a spark post then link it from the docs?  the docs are kind of the canonical place for how the system works.  either are ok with me